packages/sdks/core-js-sdk/src/sdk/index.ts

```diff
@@ -63,10 +63,19 @@ export default (httpClient: HttpClient) => ({
    * @returns The updated authentication info (JWTs)
    */
   refresh: withOptionalTokenValidations(
-    (token?: string, queryParams?: { [key: string]: string }) =>
-      transformResponse<JWTResponse>(
-        httpClient.post(apiPaths.refresh, {}, { token, queryParams }),
-      ),
+    (
+      token?: string,
+      queryParams?: { [key: string]: string },
+      externalToken?: string,
```

Comment:
Maybe we should think about this fn signature
Does it make sense to pass token & external token on the same call?

Reply:
I don't want to interleave with the generic token mechanism which has other implications on the entire Descope API

but let me think this through

Reply:
though that though and I think that sending it as the same token may lead to inferior API, the reciver need to get them as two different attributes

Reply:
What about extending the function signature to support another argument type, like an object?
Then we could do something like this
```
{ token?: string, queryParams?: { [key: string]: string } } | { externalToken?: string, queryParams?: { [key: string]: string } }
```